saleor/core/tests/test_auth_backend.py

```diff
@@ -128,29 +128,6 @@ def test_user_with_limited_permissions(
     assert set(user_permissions) == set(limited_permissions)
 
 
-def test_user_doesnt_have_permissions_from_token(staff_user, app, rf):
```

Comment:
I've dropped this test as we create jwt tokens on our side, so we can trust them.  This test was checking an additional checker for confirming that `create_access_token_for_app` doesn't assign more permission than user & app has.